app/javascript/src/config/selectors.js

```diff
@@ -13,6 +13,7 @@ const getSelectedTechnicalAreaId = (state) => state.ui.selectedTechnicalAreaId
 const getSelectedActionTypeOrdinal = (state) =>
   state.ui.selectedActionTypeOrdinal
 
+const getPlan = (state) => state.plan
```

Comment:
I can see that this new function is dead simple but still would be nice to have a basic test against it.

Reply:
tested